### DISCUSSION TITLE
Should Hex bolt be merged with version 2.0?

### DISCUSSION THREAD

**Discussion by doudar** [doudar/SmartSpin2k#275] [General]
@marklogan17 @chinasaur @MarkusSchneider @fireyeti @bau-sec @eMadman  

Any opinions on if the hex bolt shaft option should or shouldn’t be the default (only) shaft on hardware versions going forward? Or should we keep it as an option?

It adds another few (easy to acquire parts) and increases the required assembly time, but greatly improves the robustness of the SS2K.

**Comment by marklogan17** [comment]
Just had a look at it - seems pretty cool.  I think having it as default makes sense.

**Reply by doudar** [reply]
> Just had a look at it - seems pretty cool. I think having it as default makes sense.

Is 5/16” bolt easy to acquire in Germany? I think 8mm bolt may work as well but I haven’t tried it.

Edit : I think I’ve mistaken you for the other Mark that’s in Germany so sorry if you don’t have input on the 8mm bolt question.

**Reply by marklogan17** [reply]
> > Just had a look at it - seems pretty cool. I think having it as default makes sense.
> 
> Is 5/16” bolt easy to acquire in Germany? I think 8mm bolt may work as well but I haven’t tried it.
> 
> Oh wait, I think I’ve mistaken you for the other Mark that’s in Germany :)

LOL.  I was just coming here to say that I'm not up to date on German hardware availability :)

**Comment by eMadman** [comment]
I feel like it should be the default.  It took some force to break the ABS one I printed last week.  With the steel bolt, it's a non-issue. 

M8's might have different head measurements over the imperial.  I'm only using 5/16" because I couldn't find the metric ones locally.

**Comment by fireyeti** [comment]
I like the hex built as default. Purely from a long term wear stand point. Great design!
